code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -778,4 +778,17 @@
 
 /obj/item/storage/box/fountainpens
 	name = "box of fountain pens"
-	starts_with = list(/obj/item/pen/fountain = 7)
\ No newline at end of file
+	starts_with = list(/obj/item/pen/fountain = 7)
+
+/obj/item/storage/box/aggression
+	name = "box"
+	desc = "It's just an ordinary box."
+	description_antag = "This box contains various implants that will make their owners increasingly aggressive."
+	max_storage_space = 12
+	starts_with = list(/obj/item/implantcase/aggression = 6, /obj/item/implanter = 1, /obj/item/implantpad = 1)
+
+/obj/item/storage/box/encryption_key
```

Comment:
It should grant a extra channel, but it should not allow to listen into other channels as that would be a rather unlevel playing field.